package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "bedrock-web-store",
+  "version": "0.1.0",
+  "description": "Shared storage for Bedrock Web Apps",
+  "main": "index.js",
+  "dependencies": {
+    "pure-uuid": "^1.5.2"
```

Comment:
Is there something wrong with the uuid dep we've been using?  Here's a usage example:  

https://github.com/digitalbazaar/veres-delta/blob/cm4/package.json#L101
https://github.com/digitalbazaar/veres-delta/blob/cm4/components/ledger-express-config/ledger-creator-component.js#L5

Aside from consistency, the API for this new lib is nasty.

Reply:
We actually don't need any UUID dependency here I don't think. But, yes, there is a problem with the UUID dep we've been using. It only works on node -- or its "browser" version is broken/difficult to use in some fashion, likely in its build mechanism.